src/sql/src/session/vars.rs

```diff
@@ -1113,6 +1113,10 @@ feature_flags!(
         "`WITHIN TIMESTAMP ORDER BY ..`"
     ),
     (enable_managed_clusters, "managed clusters"),
+    (
+        enable_webhook_sources,
+        "creating a or pushing data to, a webhook source"
```

Comment:
This sentence feels off to me (my undergrad is in English, sorry)--would you be open to something like:

> creating or pushing data to webhook sources

Reply:
No worries at all! English class was never my strongest 😁 It's definitely a bad sentence, I updated it to your suggestion!